src/wp-admin/load-styles.php

```diff
@@ -16,6 +16,7 @@
 define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
 
 require ABSPATH . 'wp-admin/includes/noop.php';
+require ABSPATH . WPINC . '/compat.php';
```

Comment:
Doesn't make sense to include a whole file just for this. `load-styles.php` is designed to run outside of WordPress.